tests/plugins/stylescombo/langattribute.js

```diff
@@ -0,0 +1,67 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: stylescombo,toolbar */
+
+function getLangAttributeFromPanelElements( listBlock ) {
+	var els = listBlock.element.find( 'a' ),
+		ret = [],
+		el,
+		attr;
+
+	for ( var i = 0; i < els.count(); ++i ) {
+		el = els.getItem( i ),
+		attr = el.getAttribute( 'lang' );
+
+		if ( attr !== undefined ) {
+			ret.push( attr );
+		}
+	}
+
+	return ret;
+}
+
+bender.test( {
+	'test adds lang attribute to Styles combo box item': function() {
+		bender.editorBot.create( {
+			name: 'cb_1',
+			config: {
+				stylesSet: [
+					{ name: 'testName1', element: 'strong', language: 'pl' },
+					{ name: 'testName1', element: 'strong', language: 'en' },
+					{ name: 'testName1', element: 'strong', language: 'de' }
+				]
+			}
+		}, function( bot ) {
+			var editor = bot.editor,
+				stylesCombo = editor.ui.get( 'Styles' );
+
+			stylesCombo.createPanel( editor );
+
+			var lang = getLangAttributeFromPanelElements( stylesCombo._.list );
+
+			assert.areSame( 'pl', lang[ 0 ] );
+			assert.areSame( 'en', lang[ 1 ] );
+			assert.areSame( 'de', lang[ 2 ] );
+		} );
+	},
+
+	'test adds the default editor lang code to the Styles combo box item if it is not defined': function() {
```

Comment:
This test fails in my browser:

```
Values should be the same.
Expected: en (string)
Actual:   en-gb (string)
```

It would probably be better to set the language to one with a low probability of usage in our team and one different than [`config.defaultLanguage`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-defaultLanguage) – that's how we will have the certainty that the fallback really works.